packages/lab/src/carousel/Carousel.tsx

```diff
@@ -0,0 +1,191 @@
+import { Button, makePrefixer } from "@jpmorganchase/uitk-core";
+import {
+  ChangeEventHandler,
+  Children,
+  forwardRef,
+  HTMLAttributes,
+  ReactElement,
+  useEffect,
+  useState,
+} from "react";
+import { RadioButtonGroup } from "../radio-button";
+import { ChevronLeftIcon, ChevronRightIcon } from "@jpmorganchase/uitk-icons";
+import warning from "warning";
+import { useId } from "../utils";
+import cx from "classnames";
+import "./Carousel.css";
+import {
+  LayoutAnimationDirection,
+  LayoutAnimationTransition,
+} from "@jpmorganchase/uitk-core/src/layout/types";
+import { DeckLayout } from "../layout";
+
+const withBaseName = makePrefixer("uitkCarousel");
+export type SlideDirections = "left" | "right";
+
+export interface CarouselProps extends HTMLAttributes<HTMLDivElement> {
+  /**
+   * The active Index enables the consumers to select the active slide in the carousel.
+   * Optional, default 0.
+   **/
+  activeIndex?: number;
```

Comment:
This looks like a controlled prop, but when passed to `useSlideSelection`, it becomes uncontrolled. Hence raise the question - if user want to control it, how?

Reply:
it should probably be called `initialIndex`